test/gateway/eip20_gateway/revert_stake.js

```diff
@@ -52,9 +66,254 @@ contract('EIP20Gateway.revertStake()', function (accounts) {
       burner,
     );
 
+    let hashLockObj = Utils.generateHashLock();
+
+    stakeMessage.hashLock = hashLockObj.l;
+    stakeMessage.unlockSecret = hashLockObj.s;
+    stakeMessage.messageHash = messageBus.messageDigest(
+      stakeMessage.intentHash,
+      stakeMessage.stakerNonce,
+      stakeMessage.gasPrice,
+      stakeMessage.gasLimit,
+      stakeMessage.staker,
+      stakeMessage.hashLock,
+    );
+
+    await gateway.setStake(
+      stakeMessage.messageHash,
+      stakeRequest.beneficiary,
+      stakeRequest.stakeAmount,
+    );
+    await gateway.setStakeMessage(
+      stakeMessage.messageHash,
+      stakeMessage.intentHash,
+      stakeMessage.stakerNonce,
+      stakeMessage.gasPrice,
+      stakeMessage.gasLimit,
+      stakeMessage.staker,
+      stakeMessage.hashLock,
+    );
+
+  });
+
+  it('should emit event on success', async function () {
+
+    let penalty = new BN(150);
+
+    await baseToken.approve(gateway.address, penalty, {from: stakeMessage.staker});
+    await gateway.setOutboxStatus(
```

Comment:
new line.

Reply:
why?

Reply:
I think we all should follow some styling guide. I add new lines in my code. 
I am fine with not removing it, but we all should follow the same. 👍